.gitignore

```diff
@@ -10,8 +10,6 @@
 # https://github.com/github/gitignore/tree/master/Global
 
 artifacts
-/bin
-/bin.*
```

Comment:
Definitely don't want to remove this line, has already broken CI. 

I still think that un-`gitignore`ing `bin` will not work (I tried it and couldn't get CI to pass), but we'll see if you can make it work?

Reply:
Reverted the second line. `bin`'s internal contents are all ignored, save for `bin/dev`, do you remember what caused failures? Let's see what CI shakes out though. If it's too cumbersome, I'm happy to move _this_ script back up to the top-level (and not having any bin/dev binaries, just bin/dev-version/* ones).

Reply:
Moved it back.